File: java/src/main/java/ml/wonwoo/springdatashowcase/support/Currencies.java
package ml.wonwoo.springdatashowcase.support;

import org.javamoney.moneta.Money;

import javax.money.CurrencyUnit;
import javax.money.Monetary;
import javax.money.MonetaryAmount;

public class Currencies {

    public static final CurrencyUnit USD = Monetary.getCurrency("USD");

    public static final MonetaryAmount ZERO = Money.zero(USD);

}

File: kotlin/src/main/kotlin/ml/wonwoo/springdatashowcase/support/Currencies.kt
package ml.wonwoo.springdatashowcase.support

import org.javamoney.moneta.Money
import javax.money.CurrencyUnit
import javax.money.Monetary
import javax.money.MonetaryAmount

val USD: CurrencyUnit = Monetary.getCurrency("USD")

val ZERO: MonetaryAmount = Money.zero(USD)